tools/codegen/gen_phf_descriptions.py

```
@@ -0,0 +1,160 @@
+
+def main() -> None:
+    import yaml
+
+    targets = (
+        "load_phf_pbp",
+        "load_phf_player_boxscores",
+        "load_phf_schedules",
+        "load_phf_team_boxscores",
+    )
+    schemas = yaml.safe_load(pathlib.Path("tools/codegen/schemas/loader_schemas.yaml").read_text(encoding="utf-8"))
+    out, unparsed = {}, []
```

Flake8 (7.3.0)

[error] local variable 'unparsed' is assigned to but never used

(F841)